pkg/client/connector/install.go

```diff
@@ -1030,6 +1030,8 @@ func (ki *installer) managerDeployment(env client.Env, addLicense bool) *kates.D
 	if env.SystemAPort != "" {
 		containerEnv = append(containerEnv, corev1.EnvVar{Name: "SYSTEMA_PORT", Value: env.SystemAPort})
 	}
+	clusterID := ki.getClusterId(c)
+	containerEnv = append(containerEnv, corev1.EnvVar{Name: "CLUSTER_ID", Value: clusterID})
```

Comment:
I'm open to better ideas about how to get the cluster ID known by the agent, my first thought was to have the agent query it directly, but that would require a service account for the agent to use the kubernetes api, which seemed... pretty heavy / difficult for users? 

But if that's something we think would be more ideal + handled w/ documentation, let me know!